### PR TITLE
🐛 bug/98 update nickname in header when profile changes

### LINKED ISSUE #98: 🐛 Edit profile nickname does not update profile name in header without a refresh
## Description

edit profile nickname does not update profile name in header without a refresh

## Reproduce

Steps to reproduce the behavior

1. Go to 'any page'
1. Click on 'profile name / email' in top right hand corder
1. Edit profile and change the nickname (and/or email ?) 
1. You will be redirected to home page
1. The top right hand corner maintains the original name unless you refresh the page

## Expected behavior

Ideally the name in the right hand corner would update after changing the profile

### PR DESCRIPTION
**Pull request by house9**
## Story

Fix #98 

## Description

edit profile nickname does not update profile name in header without a refresh

## Analysis and Design

Need to refetch the current user when profile changes

## Solution

use `useAuth` `reauthenticate` method after updating the profile

Also:

- added `trace on` to `test:e2e` scripts
- added `test:e2e:debug` to scripts
- add seeding to playwright CI step
- removed Safari from playwright configuration

## Affected Areas

Profile Edit

## Tests

Added e2e test to verify 

## Was this feature tested on all browsers?

- [x] Chrome
- [x] Firefox

## Definition of Done

- [x] Tests written
- [x] Acceptance criteria implement
- [x] Code reviewed

## PR Comments Emoji Legend

😻 - `:heart_cat_eyes:` Compliment to code/idea/etc
♻️ - `:recycle:` Non-blocking proposed refactor
➕ - `:heavy_plus_sign:` Non-blocking proposed addition
🔥 - `:fire:` Non-blocking proposed removal
❓ - `:question:` Non-blocking question
⚠️ - `:warning:` Blocking comment that must be addressed before PR

## Gifs for life (required)

mandatory

### CODE REVIEW

**Review by iameddieyayaya — APPROVED** [review]
LGTM 💰

File: web/tests/profile-edit.spec.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  await page.locator('input[name="username"]').click()
+  await page.locator('input[name="username"]').fill('admin@example.com')
+  await page.locator('input[name="password"]').click()
+  await page.locator('input[name="password"]').fill('password')
+
+  await page.locator('button:has-text("Login")').click()
+  await page.waitForURL('/')
+
+  const logout = await page.locator('[data-testid="logout"]')
```
Comment (craineum) [inline web/tests/profile-edit.spec.ts:20]: ❓ Couldn't you have used the `Logout` text to find this instead of adding a `data-testid`?

**Review by craineum — COMMENTED** [review]
One minor question, but otherwise looks great. Have we started to look into why it is failing on CI?

**Comment by craineum** [comment]
@archae0pteryx Not sure if PR #84 will help resolve this, but I imagine there may be conflicts between them?

**Comment by archae0pteryx** [comment]
> @archae0pteryx Not sure if PR #84 will help resolve this, but I imagine there may be conflicts between them?

quite possibly. I will check it out here. I know that we've been having issues getting playwright to work correctly with safari already so that e2e fail is because of that.

**Pull request merged**